### ISSUE TITLE
Model Selection

### ISSUE THREAD

**Issue by bicheng-xu** [Scarecrow0/SGTR#6]
Hi,

Thanks for the great work! I have downloaded your pretrained checkpoint (sgtr_vg_new_pth) on the VG dataset, and found out from the log.txt that you have trained the network for 115200 iterations. However, the saved model you provide to us is model_0095999.pth. May I ask how do you select out the model_0095999.pth from all saved models?

[Checkpoint Link (VG)](https://shanghaitecheducn-my.sharepoint.com/personal/lirj2_shanghaitech_edu_cn/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Flirj2%5Fshanghaitech%5Fedu%5Fcn%2FDocuments%2Fsgtr%5Fckpt%2Fsgtr%5Fvg%5Fnew%5Fpth%2Ezip&parent=%2Fpersonal%2Flirj2%5Fshanghaitech%5Fedu%5Fcn%2FDocuments%2Fsgtr%5Fckpt&ga=1)

Thanks!

**Comment by Scarecrow0** [comment]
Hi, 
Thanks for interest in our work at SGTR.

Our model is evaluted on validation set of VG during the training.  I simply select the model which achives best val performance as final model.
